programs/test/ssl_cert_test.c

```diff
@@ -81,10 +81,21 @@ const char *client_private_keys[MAX_CLIENT_CERTS] =
 int main( void )
 {
     int ret, i;
+#if defined(MBEDTLS_PLATFORM_C)
+    mbedtls_platform_context platform_ctx;
+#endif
     mbedtls_x509_crt cacert;
     mbedtls_x509_crl crl;
     char buf[10240];
 
+#if defined(MBEDTLS_PLATFORM_C)
+    if( ( ret = mbedtls_platform_setup( &platform_ctx ) ) != 0 )
+    {
+        mbedtls_printf( " failed\n  !  mbedtls_platform_setup returned %d\n\n", -ret );
```

Comment:
We're displaying a negative return value here, yet at line 227, are displaying it without the negative. We should be self-consistent within the same program.